contrib/release/pull-docker-manifests.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2020 Authors of Cilium
+
+DIR=$(dirname $(readlink -ne $BASH_SOURCE))
+source $DIR/lib/common.sh
+
+CONTAINER_ENGINE=${CONTAINER_ENGINE:-docker}
+IMAGES=(cilium clustermesh-apiserver docker-plugin hubble-relay operator operator-generic operator-aws operator-azure)
```

Comment:
Backporters, For backports to v1.7/v1.8, we need to trim back these image lists. See the `.github/ISSUE_TEMPLATE/release_template.md` for a list of images per branch version.